src/Datasource/Connection.php

```diff
@@ -22,9 +22,9 @@
 use Cake\ElasticSearch\Exception\NotImplementedException;
 use Cake\Log\Log;
 use Elastica\Client as ElasticaClient;
-use Elastica\Log as ElasticaLog;
```

Comment:
This class doesn't actually exist anymore. I used a null logger so we don't consume memory or IO.